packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -127,6 +127,10 @@ abstract class FlutterDriver {
   /// `isolateNumber` is set, as this is already enough information to connect
   /// to an isolate.
   ///
+  /// `headers` optionally specifies HTTP headers to be included in the
+  /// WebSocket connection. This is only used for VMServiceFlutterDriver
```

Comment:
```suggestion
  /// [WebSocket] connection. This is only used for [VMServiceFlutterDriver]
```

to get cross links in our dart docs.